CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+v3.1.5
```

Comment:
```suggestion
v3.1.6
```

Reply:
I was confused as to why the `module.yml` file mentions `version: 3.1.6.dev1646404630`
Is it because of the `module.yml` file that I should put v3.1.6 and not v3.1.5 ?
Also is there a reason why the micro semver number is not incremental ?

Reply:
The source of the confusion here is that no changelog entry was added for `v3.1.5`. That's why the changelog now needs to jump from `V3.1.4` to `v3.1.6`. In general it's safe to use the `X.Y.Z` version number in the `module.yml` file generated by the `inmanta module commit` command. The `module.yml` file will also have the `.devxxxx` part in the version number. When you commit these changes on the master branch using the `inmanta module commit -r` command, the `.devxxxx` part will be removed. In the changelog you never need to mention the `.devxxxx` part as the changelog is managed entirely by hand.